### PR TITLE
Bump med stack sizes to standard

### PR DESCRIPTION
**Pull request by metalgearsloth**
Most things are size 30 so this just stops you carrying 100x healing items in an average backpack

:cl:
- tweak: Med item sizes changed from 1 to 3 so they standardise at 30 per stack.

### CODE REVIEW

**Comment by LankLTE** [comment]
Medical belts are intended to hold a lot of meds, doesn’t this completely fuck them over? (And also breaks them because they now hold like 160 worth of stuff)

**Comment by metalgearsloth** [comment]
> Medical belts are intended to hold a lot of meds, doesn’t this completely fuck them over? (And also breaks them because they now hold like 160 worth of stuff)

Thought we had a test for it

**Comment by LankLTE** [comment]
> Thought we had a test for it

I mean probably idk, but I know that this will probably break medical belts and also every medkit and also the medical bundle because it's all balanced around  them weighing 10u. 
I get the idea behind wanting to stop them from being carried around in everyone's bags, but this is a really bad way to go about fixing that imo because it's also severely nerfing medics (including nukies) and also doesn't make a ton of sense (10 bandages weighing the same as a stack of steel / 1 bandage weighing more than 3).

**Comment by metalgearsloth** [comment]
Carrying 100 meds also doesn't really make sense either and I care more about the balance than logistics.

**Comment by JoeHammad1844** [comment]
this is the worst idea i have ever seen in ss14 pull request history. completely illogical and intense reverse power creep against everyone. this doesnt benefit anyone at all and just makes everyone's lives 3x worse.

this makes the medical items basically fucking useless as the downsides would be too big to even consider them and makes liquid meds even more of a necessity than they are now.

with this change you could hold THREE stacks of medical items, four with a duffel bag.

**Comment by JoeHammad1844** [comment]
30 size should not be the standard at all, its just too big

**Comment by metalgearsloth** [comment]
> this is the worst idea i have ever seen in ss14 pull request history.

Personally I think it was IEntity but opinion disregarded.

**Comment by DogZeroX** [comment]
No, it was melee refactor.

**Comment by JoeHammad1844** [comment]
> Personally I think it was IEntity but opinion disregarded.

never heard of it :trollface:

**Comment by Endecc** [comment]
Size 30 for stackable items is to my knowledge, only standard for Raw Materials. Steel, Silver, Glass, You know, the stuff you can make **whole walls and machines** out of, and **stack to 30**. Another interpretation of a "Standard" here is that stackable items weigh One, no more or less.

Even if you want to ignore the _"logistics"_ of it, from a balance perspective, being able to heal damage in a convenient stackable form is very important for roles like the Paramedic, or Nukie medic, to agree with Lank here. Additionally, self-care with topicals is quite slow, so it's not like you're preventing John Syndicate from tiding across the whole station just because he brought more brute packs.

This balance change is gutting far more appropriately tuned aspects that don't need adjustment, than it is nerfing what it needs to. If there's a better solution, find it. Otherwise this does not need merging.

**Comment by arimah** [comment]
I dislike this for a number of reasons, most of which have been pointed out already:

* Everything else is balanced around topical meds weighing 1 per unit:
    * With this change, every single medkit is now overfilled by at least 20, in some cases considerably more.
    * Every medical belt is also overfilled by a factor of approximately three. (Except the paramed's belt, which has mostly medipens in it. That's only _slightly_ overfilled.) Medical belts are intended to hold _lots_ of meds, not a tiny amount of meds.
* Topical meds are typically needed in large quantities to fix up the dead before resuscitation. This change means you now need to run back and forth between your patients and the NanoMed or medfab to restock, or you leave things in the open where tiders can (and will) waste your topicals by self-medicating.
* It cripples field medics for nukies. Especially as nukie planet chem has been nerfed (#17518), so nukie medics can't easily make liquid chems to make up for this shortfall. I say this as someone who doesn't even play nukie but still wants them to stand a chance to win occasionally.

I feel like "30 per stack is standard" is a poor argument that does not take into account gameplay needs, nor the seeming illogic of one small tube of ointment weighing the same as one entire sheet of steel. (If anything, materials could be made heavier, but that's a separate matter. (It also doesn't make sense that a paper weighs the same as a tube of ointment, nor that a piece of paper is 1/3 the weight of a steel sheet, but this is a _game_.))

Basically, severe reverse power creep, unable to see how this _actually_ benefits anyone in any reasonable way.

**Comment by Endecc** [comment]
> you guys are gonna lose it when we refactor to storage to item slots and storage capacity gets reduced across the board by 90%

A complete refactor is another pill to swallow entirely, and comes with plenty more balancing concerns across the entire game than just medical. I'm more comfortable seeing how that plays out in the future, than a PR like this that's gutting topicals in the current system we have.

I think you're comparing a single apple to a full basket of oranges, here.

**Comment by LankLTE** [comment]
> you guys are gonna lose it when we refactor to storage to item slots and storage capacity gets reduced across the board by 90%

I’d rather it be across the board that one specific thing that’s barely even a problem

**Comment by LordEclipse** [comment]
I'm just imagining brute packs as massive as a pile pf steel plates now.

**Comment by deltanedas** [comment]
whats the actual issue here, no doctor runs around with 10 stacks of bruise packs (or can afford to)
if this is just about tiders putting meds in their bags they can already do this:
- they can just take bicar pills
- they can just take bicar jugs
- they can just take medfab parts and mats to make bruise packs since thats better density

**Comment by QuietlyWhisper** [comment]
Youre going to also have to go fix all belt and medkit fills.

While I do agree carrying a shit ton of these is silly, youll likely have to increase their healing because both nukeops and doctors(which is not as bad honestly) will not be able to carry enough healing to be useful, as even if you cut down to 3 or 5 packs between ointments, bruise packs, and blood packs, youll get 25 healing at most per type which in most cases isnt enough to get a player back into the game without chemicals.

These items are the basic essentials of healing. Doctors should be fine with this nerf besides having to walk back and fourth a lot, and it will cripple a lot of tiders that carry meds for no particular reason, but it will also heavily cripple antags that do not have access to the amount of medical supplies the station does, be it because of no access to chems, or no access to basic healing.

Other than that, seeing less supplies go missing because thet take up too much space to carry around would be great

**Comment by JoeHammad1844** [comment]
> you guys are gonna lose it when we refactor to storage to item slots and storage capacity gets reduced across the board by 90%

so ss13 storage? thats fine, that system is pretty good, robust even

**Comment by JoeHammad1844** [comment]
> but it will also heavily cripple antags that do not have access to the amount of medical supplies the station does, be it because of no access to chems, or no access to basic healing.

im going to bring it up before someone else does, you can buy a combat medkit, but that is a quarter of your TC and that just fucking sucks. will basically make loud antagging impossible because you are inevitably going to take a hit and any attentive secoff will see you in all your stolen gear and contraband walk into medbay

this change will basically carpet bomb everyone, and i dont see whats wrong with meds being portable since you can just get infinitely more from the medfab. no materials? ask cargo. no money? miners. no miners? hire some. no power? you probably have something worse to worry about, call the shuttle

 there are like three different safety nets that you can fall back on in case of low med packs and if none are there that means something is seriously wrong with the station and it wont be recovering anyway

**Comment by deltanedas** [comment]
> Other than that, seeing less supplies go missing because they take up too much space to carry around would be great

just dont leave 5 medkits lying around in the med lobby easy, or give medics telebatons to bonk tiders with

**Comment by JoeHammad1844** [comment]
giving heads telebatons actually isnt a bad idea, the entire purpose is for self defence and to protect the department until sec can get there anyway. sad the PR that was going to add it was abandoned.. unless?

**Comment by deltanedas** [comment]
time for you to revive so cmo can defend medicals stock :trollface:

**Comment by arimah** [comment]
> youll likely have to increase their healing

... at which point you have to consider how that affects the balance in other ways, as it might now be worth the sacrifice to carry 6 bruise packs if a single one can heal, say, a third of your damage – and faster than before, too, unless you also increase the doafter time. And if they get more powerful, you may additionally wish to rethink the material cost in the medfab. And advanced versions (sutures, mesh) would likely have to heal _even more_[^1], at which point John Syndicate or Jane Tide may as well steal those instead and recover from most damage in no time.

[^1]: or heal the same but faster, or some other balancing act to justify their existence

Yes, the above is all hypothetical and the healing potential is untouched by this PR. But game balance is hard. The current balance is not flawless, but there are many ways of inadvertently making things worse, and this PR seems primarily to do that.

**Comment by QuietlyWhisper** [comment]
This whole thing would also probably be a lot less of an issue if tend wounds surgery existed, which requires an assisting player with the correct tools rather than two stacks of bandaids

It could honestly probably be added without affecting the wounds refractor assuming that wounds don't carry their own damage and are simply debuffs, but thats a bit of a tangent and I havent read the doc in a while

**Comment by GoodWheatley** [comment]
If this is about nerfing lone wolves who carry all the medical supplies they need in their backpack, then this is *already* nerfed by the code increasing the doafter time and reducing the healing if you’re using them on yourself. 

This seems extremely unnecessary and is going to have a ton of unintended side effects that you just seem to be glossing over for the sake of “consistency”.

**Comment by QuietlyWhisper** [comment]
> This whole thing would also probably be a lot less of an issue if tend wounds surgery existed, which requires an assisting player with the correct tools rather than two stacks of bandaids
> 
> It could probably be added without affecting the wounds refactor assuming that wounds don't carry their own damage and are simply debuffs, but thats a bit of a tangent and I havent read the doc in a while

Talked to Jez and she said basic tend wounds in current state is a waste of time bc damage is handled in a very different way

**Comment by metalgearsloth** [comment]
> whats the actual issue here, no doctor runs around with 10 stacks of bruise packs (or can afford to)

Antags run around with 5 stacks, plus other combat items, and still having 30 free space left over, hence the issue.

**Comment by Subversionary** [comment]
30 storage to heal 50 brute/burn over a minute or two is too much.

Since the crux of most complaints is sosa robust cookie healing brute damage with sutures or bruise packs whenever possible just introduce damage and slow on user is his self-heal is interrupted by any mean.

And healing while moving in no-grav interrupts said doafter because its obscenely broken.

**Comment by EmoGarbage404** [comment]
> 30 storage to heal 50 brute/burn over a minute or two is too much.

Well, first of all, it's 150 healing, not 50. 50 healing is still 10 storage, funnily enough. 

It's not a minute, either. It takes only 9 seconds to heal 45 damage. Even with the worst penalties with self-healing, it's still going to be roughly 25 seconds to heal off 45 damage, and it only gets faster if you have less damage.

Healing is incredibly effective and quick, and the fact that you can buy them from cargo and print them en masse only makes them more powerful.

Making healing items have greater size just makes it so that you actually have to centralize medical equipment inside of the treatment areas, instead of being walking hospitals carrying enough supplies to heal a small battalion.

**Comment by Subversionary** [comment]
> Well, first of all, it's 150 healing,

>  It takes only 9 seconds to heal 45 damage

> Healing is incredibly effective and quick

This assumes the target has 50 blunt, slash and pierce at once, selfheal is not an issue here as user would be in crit. This is max potential of a bruise pack and its rarely reached. In the scenario which raised this issue (soloop/tot sosa negating damage after a firefight) its predominately pierce damage, moreso after the bow existing.

Anyone with access to a jet, eva, space and sutures can literally go “fuck you and your pressure” because he is healed 10 pierce after each doafter.

**Comment by LordEclipse** [comment]
How does this not incentivize people to just carry around bottles of chems and pill bottles?

**Comment by Aisu9** [comment]
what about blacklisting any suture/meshes/packs/ointment from everything things BUT kits-medical belt-fanny pack and blacklist every types of medkits-medical belt-fanny pack from backpacks?
this would significantly reduce the amount of meds one can carry on them

**Comment by rbertoche** [comment]
Isn't it a bit awkward that you can't fill a bag with bruise packs? I mean, that stuff does seem small after all, why would it only fit 3 boxes of it in a whole empty backpack?
If you can print them too easily that's a whole different issue, it is a bit weird after all you don't need chemicals to produce those

**Comment by Kromabae** [comment]
who is this change for?

**Comment by NancokPS2** [comment]
While it may SEEM ralistic, this destroys the balance and fun of healing as a whole.    

Bruise packs and other small medical equipment heals for pitiful amounts. Imagine carrying around a backpack FULL of medical items, and emptying it entirely yet not being able to make someone revivable (since each basic healing item is like 5 damage per item).    
  
And i don't even want to know what will happen to belts.    
It would be nice to know what was the actual reason behind this decision other than "I personally don't think people should be able to carry this amount of meds".

**Comment by JoeHammad1844** [comment]
i agree, the current balance does not need to change until wounds and/or surgery are added

**Comment by deltanedas** [comment]
- pr description is a non reason, why not change biomass to 30 max  as well so its like steel
- if its just to nerf sosa increase self heal time even more

**Comment by Aisu9** [comment]
> * pr description is a non reason, why not change biomass to 30 max  as well so its like steel
> 
>     * if its just to nerf sosa increase self heal time even more

let me remind you that sosa alone was the reason for some stupid feature gutting when some changes and basic roleplay could have fix them

**Comment by JoeHammad1844** [comment]
> let me remind you that sosa alone was the reason for some stupid feature gutting when some changes and basic roleplay could have fix them

specific examples please?

**Comment by deltanedas** [comment]
desword

**Comment by JoeHammad1844** [comment]
elaborate

**Comment by deltanedas** [comment]
#18132

iirc direct response to sosa ops

**Comment by JoeHammad1844** [comment]
oh bruh

**Comment by JoeHammad1844** [comment]
most people arent sosa, we shouldn't balance the game around them

**Comment by NancokPS2** [comment]
You mean that the game about RP and paranoia shouldn't be balanced around combat? What a controversial idea (apparently)

**Comment by keronshb** [comment]
> #18132
> 
> iirc direct response to sosa ops

Can you explain what sosa is.

**Comment by deltanedas** [comment]
valid

**Comment by QuietlyWhisper** [comment]
> Can you explain what sosa is.

Tldr; Strong player, posts are accusing this pr being a grudge to nerf them, pr doesnt address issues that may come up from the changes

If you are actually out of the loop, they're one of the more mechanically inclined players of the lrp playerbase

Theyre a fairly big outliner in the playerbase, though. They're typically able to figure out how to get an edge over a hostile player in any way possible.

Essentially these posts on the pr are accusing the changes of simply being to "nerf" the particular player with no other reasoning. I assume much like how most competitive games balance things around their top players and tournaments. 

But in this particular case, it does make sense for these items to take up more space, but at the same time they are hardly worth carrying as is. There are complaints because this pr doesn't appear to address the entire scope of how it will affect the game.

It was pretty clear cut for materials because its like oh you can build 15+ walls with two stacks of steel, thats not needed and dumb.

In this case though the healing items are balanced to be weaker than chemicals and proper full treatments, which is why so many are needed. This doesnt even take into account that several medkits and the medical belts need to be refilled and/or rebalanced.

**Comment by lexs0sa** [comment]
As someone mentioned, the current materials sit at 3 per item which consistently is far off from something as small as a brute pack or ointment. And while I appreciate the praise(?) and not to derail, desword nerf was not made because of me, multiple teams were running full on desword nightmare mode campaigns, I actually never used it. Stims, on the other hand...

I will say, before going loud (as I tend to do, call me kooky) I do often get an absolutely ludicrous amount of meds on my person, I think this would genuinely make people think twice about engagements and have to actually think out what they need in their storage. As it stands you can hold 20 sutures, 20 regen meshes, a Lecter and two magazines just with a regular backpack, coupled with a jetpack and proper access that's an absolute nightmare to have to deal with - moreso on any station that has external gravity and armory access (surprise, most do.) 

I personally welcome it, though 2 weight might be more generous for regular users. Holding a 10/10 stack of Sutures and Regen meshes being 60 weight is punishing, but again - the points made here to support the PR is that you *should be in Medical* while receiving treatment, which I have to agree with. I don't know how this will affect Nukies, but they should be using chems primarily (and the hypospray) for treatment anyway, meds are a superfluous preparation for other operatives if the Agent is aware of what they're doing.

**Comment by rbertoche** [comment]
The way I see it for most people reading it the fact this PR is even submitted is concerning, since it had real chances of getting merged hadn't people talked about it. Had it got merged faster or when less people were looking it would probably be worse. It's hard to understand the reasoning between what's a good well justified change from reading this one.
I hope I don't sound like I'm trying to be negative though.

**Comment by JoeHammad1844** [comment]
adding this would mean that medbelts would need to get even bigger, but by then topicals wont be worth it at all so people will just carry around like 20 bottles with various meds inside, which will just increase the amount people will be able to heal, negating the purpose of this PR.

it will also make it a lot harder to revive dead people since you need to keep the packs somewhere and they will probably just get blown up or get hidden by an antag

come to think of it, liquid meds are just better than topicals because you can just get a syringe, inject yourself and heal faster than topicals in a more compact size, topicals are heavier and heal less than liquid meds and are only taken because its just easier to grab bruise packs than it is to somehow get a bottle and syringe without asking anyone

**Comment by Subversionary** [comment]
> Essentially these posts on the pr are accusing the changes of simply being to "nerf" the particular player with no other reasoning.

The strat mentioned is complete cheese because its a consequence of how doafters on self are handled in nograv. If not for that selfheal wouldnt be an issue, but mechanically inclined players are able to move and heal at the same time. This doesnt address the “cried-about” issue, this applies a blunt object to medicine without giving a single solitary thought about chems our storage fills, as evidenced by the pull request OP.

**Comment by JoeHammad1844** [comment]
> “Robust” players will just move to pills and syringes for an even more rapid heal in exchange for using less storage.

chem bottles and syringes are more compact and more efficient, but topicals are just easier to acquire for non medical personnel

**Comment by HoidC** [comment]
> > “Robust” players will just move to pills and syringes for an even more rapid heal in exchange for using less storage.
> 
> chem bottles and syringes are more compact and more efficient, but topicals are just easier to acquire for non medical personnel

dude politely could you stop spamming this PR with counterpoints to things that have already been said 

credible responses and feedback gets weighed down when people for this kind of thing justify it to themselves by going “player base slop I know better” and are vindicated and you really aren’t helping that

**Comment by JoeHammad1844** [comment]
sure, just wasn't self aware enough to catch myself

**Comment by Subversionary** [comment]
> dude politely could you stop spamming this PR with counterpoints to things that have already been said

He does this chronically on every fucking balance PR of any significance and proposes complete powercreep in his own ones which get denied on average.

**Comment by JoeHammad1844** [comment]
sorry for providing input, bro

**Comment by JoeHammad1844** [comment]
apologizing then immediately doubling down, how smart of me :trollface:

**Comment by metalgearsloth** [comment]
https://github.com/space-wizards/space-station-14/issues/21134 is likely going to supersede this though I don't intend to work on it for at least another 6-12 months realistically.

**Comment by lexs0sa** [comment]
This would have actually gone pretty hard. As it stands you can hold 20 sutures, 20 meshes, a Lecter and 2 mags. That, with a jetpack and proper accesses, alongside external facing grav-gens and armories which, surprise surprise, most are, is a nightmare to deal with. I think patching outliers to fit more in tune with how the spessgame is intended to be played, even if the combat is an extremely integral and fun experience, will facilitate better gameplay for those who want to counter a currently extremely powerful meta. Focusing medical healing on actual medbay would do leagues better than someone being able to pop 2 epens and some sutures to completely negate a firefight, it would require potential allies and teammates for coordination, having to sneak back into medbay, etc.; personally the real downside would be that it does heavily punish non-antagonists, but when I play doctor I tend to stick to medbay and as a paramedic I epen and use rollerbeds to drag victims to medbay anyway, as they have the chems for the job. 

Also I never used desword as a nukie or Syndie - maybe once or twice as a joke, but it was extremely grossly busted to the point of not even being fun. Stims on the other hand... I digress. This would have gone hard!

**Comment by metalgearsloth** [comment]
See https://github.com/space-wizards/space-station-14/issues/21134 and the incoming pr.
